packages/react-app-revamp/config/wagmi/custom-chains/degenChain.ts

```diff
@@ -0,0 +1,24 @@
+import { Chain } from "@rainbow-me/rainbowkit";
+
+export const nautilusChain: Chain = {
+  id: 69420,
+  name: "Degen",
```

Comment:
Also name and network name need to be the same

Reply:
updated here as well.